plugins/prow-job/commands/list-jobs.md

```diff
@@ -0,0 +1,32 @@
+---
+description: list all CI jobs
+argument-hint: bearer-token keywords
+---
+
+## Name
+/prow-job:list-jobs
+
+## Synopsis
+Generate a report showing all openshift qe ci jobs:
+```
+/prow-job:list-jobs <bearer-token>
```

Comment:
IMHO the token shouldn't be used in a slash command.  Use a preset environment variable, or look at the pattern in the `ci` plugin for curling with a token.

If you ever wanted to demo this, your private token would be on display to everyone. It also shows up in logs, etc.